language/move-lang/tests/move_check/parser/address_too_long.exp

```diff
@@ -3,6 +3,6 @@ error:
    ┌── tests/move_check/parser/address_too_long.move:2:9 ───
    │
  2 │ address 0x112233445566778899101122334455667788992011223344556677889930112233:
-   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ The address [17, 34, 51, 68, 85, 102, 119, 136, 153, 16, 17, 34, 51, 68, 85, 102, 119, 136, 153, 32, 17, 34, 51, 68, 85, 102, 119, 136, 153, 48, 17, 34, 51] is of invalid length. Addresses are at most 32-bytes long
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Address is longer than the maximum size of 32 bytes
```

Comment:
It might be nice if the number of bytes parsed was given

Reply:
That's a good point. I also noticed that there is a constant ADDRESS_LENGTH value that I can use instead of hardcoding the 32 limit. I pushed an updated patch.